.drone.yml

```diff
@@ -5,19 +5,32 @@ pipeline:
       - npm --loglevel warn install -g npm@3
       - npm --loglevel warn install
       - npm run test:ci
+    when:
+      event: [push, pull_request]
 
   build_5:
     image: node:5
     commands:
       - npm --loglevel warn install
       - npm run test:ci
+    when:
+      event: [push, pull_request]
+
+  build_app:
+    image: docker:1.11
+    environment:
+      - DOCKER_HOST=tcp://127.0.0.1:2375
+    commands:
+      - docker build -t rotm .
+    when:
+      event: pull_request
 
   build_app:
     image: docker:1.11
     environment:
       - DOCKER_HOST=tcp://127.0.0.1:2375
     commands:
-      - docker build -t app .
+      - docker build -t rotm .
```

Comment:
reason why you changed this? BTW, if you change it here you should change value you tag the image has with https://github.com/UKHomeOffice/rotm/blob/3538874d3180eba77076388a4e7920f8b95143dc/.drone.yml#L93